js/Discourse.js

```diff
@@ -234,13 +234,8 @@ class Discourse extends React.Component {
     const url = e._data && e._data.discourse_url;
 
     if (url) {
-      this._siteManager.setActiveSite(url).then(activeSite => {
-        let supportsDelegatedAuth = false;
-        if (this._siteManager.supportsDelegatedAuth(activeSite)) {
-          supportsDelegatedAuth = true;
-        }
-        this.openUrl(url, supportsDelegatedAuth);
-      });
+      this._siteManager.setActiveSite(url);
```

Comment:
So this is not a promise anymore?

Reply:
Right, we no longer need to wait for the promise to complete so we can find out if it supports delegated auth. Delegated auth was added 3+ years ago, we now assume all Discourse sites have support for it.